addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -78,10 +78,22 @@
                 <ul class="list-group d-block o_search_panel_field">
                     <li t-foreach="group.dashboards" t-as="dashboard" t-key="dashboard.id"
                         t-on-click="() => this.openDashboard(dashboard.id)"
-                        t-esc="dashboard.displayName"
                         t-att-data-name="dashboard.displayName"
-                        class="o_search_panel_category_value list-group-item cursor-pointer border-0"
-                        t-att-class="{'active': dashboard.id === state.activeDashboard.id}"/>
+                        class="o_search_panel_category_value list-group-item cursor-pointer border-0 d-flex justify-content-between align-items-center"
+                        t-att-class="{'active': dashboard.id === state.activeDashboard.id}">
+                        <div class="o_dashboard_name">
+                            <t t-esc="dashboard.displayName" />
+                        </div>
+                        <button
+                            type="button"
+                            class="btn btn-link o_edit_dashboard oi oi-arrow-right"
+                            tabindex="-1"
+                            draggable="false"
+                            aria-label="Edit"
+                            data-tooltip="Edit"
+                            t-on-click.stop="() => this.editDashboard(dashboard.id)"
```

Comment:
What's the purpose of the `.stop` here ?

Reply:
When we click the edit icon for a dashboard that isn't currently active, the event will first bubble up to activate the dashboard before opening it in the spreadsheet. This results in an extra network request.